packages/chat-stateful-client/src/types/ChatMessageWithStatus.ts

```diff
@@ -17,4 +17,6 @@ export type ChatMessageWithStatus = ChatMessage & {
   status: MessageStatus;
```

Comment:
Is `TODO` in the documentation part is still needed?

Reply:
This might need a follow up. I am not sure about the context of this TODO